### DISCUSSION TITLE
Was there a change that made zelda3_assets.dat a requirement?

### DISCUSSION THREAD

**Discussion by CyberAkumaZero** [snesrev/zelda3#148] [Q&A]
I first compiled this using MSYS Mingw 64 early September. After fumbling about a bit I managed to get a zelda3.exe compiled, and it ran fine. I noticed that I only needed the zelda3.exe and SDL2.dll, with zelda.ini being optional.

I saw that the repo had been updated so I downloaded the latest version and compiled it again. However, now the game was not starting. When I double-clicked the EXE nothing happened, no error message, nothing. After trial-and-error I noticed that the exe had shrunk in size by about 700KB, and that zelda3_assets.dat is around that size.

By including zelda3_assets.dat in the folder with the exe the game runs fine again.

Was this an intentional change? I found it convenient when all that was included in the exe itself instead of having to be a separate file.

**Comment by UltraHDR** [comment]
https://github.com/snesrev/zelda3/commit/3ba33373eca644b7c4b8347fe7d43deb2676b7fb
